force-app/main/default/layouts/Case-Case %28Marketing%29 Layout.layout-meta.xml

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@youngbeanie can you delete these files? they get added by scratch orgs sometimes.